indico/modules/events/timetable/templates/pdf/_program.html

```diff
@@ -0,0 +1,324 @@
+                                <div class="session-block-wrapper">
+                                    {% if entry.type.name == 'BREAK' %}
+                                        {{ _render_break(entry, config, program_config) }}
+                                    {% elif entry.type.name == 'CONTRIBUTION' %}
+                                        {{
+                                            _render_contrib(
+                                                entry,
+                                                config,
+                                                program_config,
+                                                show_children_location
+                                            )
+                                        }}
+                                    {% else %}
+                                        {{
+                                            _render_session_block(
+                                                entry,
+                                                config,
+                                                program_config,
+                                                show_children_location
+                                            )
+                                        }}
+                                    {% endif %}
+                                </div>
+                            </section>
+                        </section>
+                        <div class="time-text time-to">
+                            {{ entry.end_dt|format_time(timezone=event.tzinfo) }}
+                        </div>
+                    </div>
+                {% endif %}
+            {% endfor %}
+        </div>
+    {% endfor %}
+{% endmacro %}
+
+{% macro _render_break(entry, config, program_config) %}
+    {% set break = entry.object %}
+
+    <section
+        id="entry-{{ entry.id }}"
+        class="session-block-title inherited-bg-color inherited-txt-color"
+    >
+        {{ _render_entry_header(entry, config, program_config) }}
+    </section>
+{% endmacro %}
+
+{% macro _render_contrib(entry, config, program_config, show_children_location) %}
+    {% set contrib = entry.object %}
+    {% set speakers = contrib.speakers %}
+    {% set subcontribs = contrib.subcontributions %}
+    {% set show_contrib_content = subcontribs or (config.show_abstract and contrib.description) %}
+
+    <section
+        id="entry-{{ entry.id }}"
+        class="session-block-title inherited-bg-color inherited-txt-color {{ 'only-round-top' if show_contrib_content else ''}}"
+    >
+        {{ _render_entry_header(entry, config, program_config) }}
+    </section>
+    {% if show_contrib_content %}
+        <div class="session-block-content">
+            {{ _render_description(contrib.description) }}
+            {% if subcontribs %}
+                {{ _render_subcontribs(contrib, config, show_children_location) }}
```

Comment:
SubContributions do not have location information, so I'd pass that last arg as `False`. I think right now you could end up showing a Location header w/o content...

Reply:
I have actually removed the argumen tin its entirety in some places and put a default value `false`